snmp_cisco/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
I think we can remove `metadata.csv` completely since it's not used by manifest.json anymore.

Reply:
Good catch, left over from the cleaning, indeed it's not needed